cmd/swapd/main.go

```diff
@@ -215,7 +215,6 @@ var (
 func main() {
 	if err := app.Run(os.Args); err != nil {
 		log.Fatal(err)
-		os.Exit(1)
```

Comment:
I don't think there are any loggers for Go that don't exit in the Fatal* methods, but I made sure the IPFS logger wasn't an exception before committing this.

Reply:
do they exit with a non zero exit code?

Reply:
Yes, the standard library `log.Fatal`, this IPFS `log.Fatal` as well as logrus' `Fatal` all exit with the status of `1`.